template_feed/Microsoft.DotNet.Common.ItemTemplates/content/BuildProps/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json.schemastore.org/dotnetcli.host",
+  "symbolInfo": {
+    "useArtifacts": {
+      "longName": "use-artifacts",
+      "shortName": ""
+    },
+    "inherit": {
+      "shortName": ""
```

Comment:
the generated short name was `-id`, which is not POSIX, so we delete it entirely instead.